backend/src/routes/v1/auth.ts

```diff
@@ -48,4 +48,64 @@ router.delete(
   authController.revokeAllSessions
 );
 
+// --- identity endpoints
+
+router.post(
+  "/token/renew", 
+  authController.renewAccessToken
+);
+
+router.post(
+  "/universal-auth/login",
+  authController.loginIdentityUniversalAuth
+);
+
+router.post(
+  "/universal-auth/identities/:identityId",
+  requireAuth({
+    acceptedAuthModes: [AuthMode.JWT]
+  }),
+  authController.addIdentityUniversalAuth
+);
+
+router.patch(
+  "/universal-auth/identities/:identityId",
+  requireAuth({
+    acceptedAuthModes: [AuthMode.JWT]
+  }),
+  authController.updateIdentityUniversalAuth
+);
+
+router.get(
+  "/universal-auth/identities/:identityId",
+  requireAuth({
+    acceptedAuthModes: [AuthMode.JWT]
+  }),
+  authController.getIdentityUniversalAuth
+);
+
+router.post(
+  "/universal-auth/identities/:identityId/client-secrets",
+  requireAuth({
+    acceptedAuthModes: [AuthMode.JWT]
+  }),
+  authController.createUniversalAuthClientSecret
+);
+
+router.get(
+  "/universal-auth/identities/:identityId/client-secrets",
+  requireAuth({
+    acceptedAuthModes: [AuthMode.JWT]
+  }),
+  authController.getUniversalAuthClientSecrets
+);
+
+router.delete(
```

Comment:
Revoking doesn't delete the secret, it sets the state to revoked = true. I think it would be better if we turned it to `POST` and updated the route to `/universal-auth/identities/:identityId/client-secrets/:clientSecretId/revoke`

Reply:
Also i don't think we need `identityId` here since we can get it from clientSecretId. If we revoke it, i think it would make the API request simpler for end users

Reply:
Will change to `POST`.